.travis.yml

```diff
@@ -44,6 +44,6 @@ matrix:
       os: linux-64
       arch: x86_64
 script:
-  -  cd conda-recipes/transformers-feedstock
+  -  cd conda-recipes/faiss-feedstock
```

Comment:
Should be faiss-gpu-feedstock (I cheated and looked at the build failure in travis :) )

Reply:
Yeah, I changed the recipe directory name at the last minute to add the `-gpu` to match the package name. :(